src/govuk/components/character-count/character-count.mjs

```diff
@@ -5,6 +5,42 @@ import '../../vendor/polyfills/Element/prototype/classList.mjs'
 import { closestAttributeValue, extractConfigByNamespace, mergeConfigs, normaliseDataset } from '../../common.mjs'
 import { I18n } from '../../i18n.mjs'
 
+/**
+ * @typedef {object} CharacterCountTranslations
+ * @property {import('../../i18n.mjs').PluralisedTranslation} [charactersUnderLimit]
+ * @property {string} [charactersAtLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslation} [charactersOverLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslation} [wordsUnderLimit]
+ * @property {string} [wordsAtLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslation} [wordsOverLimit]
+ */
+
+/**
+ * @type {CharacterCountTranslations}
+ */
+var DEFAULT_TRANSLATIONS = {
```

Comment:
No comment on the word "translations", but similar to `HEROKU_APP`, `GITHUB_TOKEN`, `PERCY_TOKEN`, `KEY_SPACE` etc I'd put the type/modifier _after_ the thing (if that makes sense 😆)

```suggestion
var TRANSLATIONS_DEFAULT = {
```